### PR TITLE
Payments page

### LINKED ISSUE #959: Payments page

### PR DESCRIPTION
**Pull request by asvae**
closes #959 

* Also added storybook

image

### CODE REVIEW

**Comment by asvae** [comment]
Add validation for cards

File: src/pages/payments/widgets/my-cards/PaymentCardList.vue [hunk 1]
```diff
@@ -0,0 +1,69 @@
+<template>
+  <div class="grid md:grid-cols-2 grid-cols-1 gap-4">
+    <VaSkeletonGroup
+      v-if="loading"
+      class="min-h-[114px] p-4 rounded-lg border border-dashed border-backgroundBorder flex flex-row items-center gap-6"
+    >
```
Comment (m0ksem) [inline src/pages/payments/widgets/my-cards/PaymentCardList.vue:6]: Should be with card title on mobile ([see design](https://www.figma.com/file/1FrFtIjHjMAV0o4t0wpYGq/Admin?type=design&node-id=369-3650&mode=design&t=l5CJlOwFf5ZTb9Eb-0))
image

File: src/pages/payments/widgets/my-cards/PaymentCardList.vue [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+            Please carefully read
+            <span class="underline">Product Terms</span> before adding your new payment card
+          </div>
+        </div>
+        <va-button class="flex-none w-full sm:w-auto" @click="showCreate = true">Add card</va-button>
+      </div>
+    </template>
+  </div>
+  <PaymentCardCreateModal v-if="showCreate" @close="showCreate = false" />
+  <PaymentCardUpdateModal v-if="cardToEdit" :payment-card="cardToEdit" @close="cardToEdit = ''" />
```
Comment (m0ksem) [inline src/pages/payments/widgets/my-cards/PaymentCardList.vue:38]:
```suggestion
  <PaymentCardUpdateModal v-if="cardToEdit" :payment-card="cardToEdit" @close="cardToEdit = undefined" />
```
image

Same in billing address

File: package.json [hunk 3]
```diff
@@ -11,7 +11,9 @@
     "prelint": "npm run format",
     "lint": "eslint --fix \"./src/**/*.{ts,js,vue}\"",
     "format": "prettier --write .",
-    "preview": "vite preview"
+    "preview": "vite preview",
+    "storybook": "storybook dev -p 6006",
+    "build-storybook": "storybook build"
```
Comment (m0ksem) [inline package.json:16]: We use bun now? Yarn.lock or bunlock must be removed

File: src/pages/admin/dashboard/Dashboard.vue [hunk 4]
```diff
@@ -1,40 +1,5 @@
-<template>
-  <div class="dashboard">
-    <dashboard-charts />
+<template>TODO</template>
```
Comment (m0ksem) [inline src/pages/admin/dashboard/Dashboard.vue:1]: Todo? Revert?

File: src/scss/main.scss [hunk 5]
```diff
@@ -14,23 +14,24 @@ body {
...
-h3,
-h4,
-h5,
-h6,
-strong,
-b {
-  line-height: 1.2;
-  font-weight: 600;
-}
+//h1,
```
Comment (m0ksem) [inline src/scss/main.scss:24]: Remove old code instead of commenting it.

**Review by m0ksem — CHANGES_REQUESTED** [review]

File: src/pages/admin/dashboard/Dashboard.vue (hunk 4 above)
Comment (asvae) [reply src/pages/admin/dashboard/Dashboard.vue:1]: No. We'll scrap Dashboard page. And it had errors, sometimes breaking vue. So I discarded it.

File: package.json (hunk 3 above)
Comment (asvae) [reply package.json:16]: It works fine, but no, we stick to npm.

**Pull request merged**